examples/compiler/05_optimization_modes.ipynb

```diff
@@ -36,7 +103,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# This function will be used to print the difference between the generated code.\n",
+    "#TODO: George -- I guess we could (and should!) move this under examples/performance/utils.py ... \n",
```

Comment:
Maybe we can exploit that with pytest at some point? I can think of several applications especially when asserting long strings ?